feat(rearch): cloud signup (#230)
GlareDB/glaredb.github.io: introduction/installation/sign-up-for-cloud.md

---
title: Sign Up for GlareDB Cloud
layout: default
nav_order: 1
parent: Installation
grand_parent: Introduction / Getting Started
---

# Sign Up for GlareDB Cloud

[GlareDB Cloud] is a fully-managed offering of GlareDB that scales with your

> workloads and teams. It is free, and easy to get started.

Reviewer: ```suggestion
workloads and teams. It is free to get started, and signing up is quick!
```

![Sign-in]

<!-- TODO: link to pages once they're written -->

> Upon signing up, an Organization and fully-managed GlareDB Deployment will be
> instantly created and ready to use with the **SQL Workspace** web-based editor.

Reviewer: ```suggestion
After signing up, an Organization with a fully-managed GlareDB Deployment are be
created and ready to use with the **SQL Workspace** editor in your browser.
```

> After signing up, an Organization with a fully-managed GlareDB Deployment are be
> created and ready to use with the **SQL Workspace** editor in your browser.

Reviewer 2: ```suggestion
After signing up, an Organization with a fully-managed GlareDB Deployment will be
created and ready to use with the **SQL Workspace** editor in your browser.
```

![Editor]

## Ready to invite your team?

> With the free tier, up to 5 teammates can be added to your Organization.

Reviewer: ```suggestion
With the free tier, up to 5 teammates can access the GlareDB deployments in your Organization.
```
